aw_qt/trayicon.py

```diff
@@ -124,22 +124,15 @@ def add_module_menuitem(module):
             if module_name != "aw-server":
                 add_module_menuitem(self.manager.modules[module_name])
 
+    def exit_dialog(self):
```

Comment:
How do the `menu.addAction("Quit ActivityWatch", exit_dialog)` lines above use this? Seems to me like it shouldn't work.

Related to my comment in the other PR.

Reply:
Hmm yeah I'll remove the exit_dialog entirely

Reply:
resolved in e61ece5